deb/after_install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Luca once commented on a different packaging PR that after install scripts shouldn't be bash scripts as then we'd need to specify `bash` as a dependency in the packaging configuration. Can you change this to `#!/bin/sh` instead?